src/hdmint/tracker.cpp

```diff
@@ -228,8 +228,11 @@ bool CHDMintTracker::HasPubcoinHash(const uint256& hashPubcoin) const
     }
 
     for (auto const & it : mapLelantusSerialHashes) {
+        CWalletDB walletdb(strWalletFile);
```

Comment:
this is really bad for performance. You need to use *one* `CWalletDB` class instance and pass it as an argument to different functions. Here it's inside the loop